contrib/completion/bash/docker

```diff
@@ -5287,7 +5287,6 @@ _docker_version() {
 
 	case "$cur" in
 		-*)
-			local options="--format -f --help"
 			COMPREPLY=( $( compgen -W "--format -f --help" -- "$cur" ) )
```

Comment:
Looks like I missed this one when reviewing https://github.com/docker/cli/pull/3578 😅

Reply:
Ah yes, of course. Thanks for spotting.